cabal-install/src/Distribution/Client/Config.hs

```diff
@@ -434,6 +434,7 @@ instance Semigroup SavedConfig where
           , installSymlinkBinDir = combine installSymlinkBinDir
           , installPerComponent = combine installPerComponent
           , installNumJobs = combine installNumJobs
+          , installUseSemaphore = combine installUseSemaphore
```

Comment:
How does this behave with two non-empty values?

Reply:
I guess the usual, the more local value takes priority?